helm/cluster-api/files/core/patches/versions/v1alpha2/clusters.cluster.x-k8s.io.yaml

```diff
@@ -2,7 +2,7 @@
   path: /spec/versions/0/schema
   value:
     openAPIV3Schema:
-      description: Cluster is the Schema for the clusters API
+      description: Cluster is the Schema for the clusters API.
```

Comment:
Looks like that for CAPI core CRDs, only the docs got tidied up little bit in v1alpha2 when added back to v1.0.x, so we should be good.

cc @fiunchinho @AverageMarcus